presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -71,10 +70,6 @@
     @Inject
     private TestHiveMetastoreClientFactory testHiveMetastoreClientFactory;
 
-    @Inject
-    @Named("databases.hive.warehouse_directory_path")
-    private String warehouseDirectory;
```

Comment:
folow up We should try to remove warehouse_directory_path property

cc @wendigo